crates/youki/src/main.rs

```diff
@@ -97,7 +97,7 @@ fn main() -> Result<()> {
     let root_path = determine_root_path(opts.global.root)?;
     let systemd_cgroup = opts.global.systemd_cgroup;
 
-    match opts.subcmd {
+    let output = match opts.subcmd {
```

Comment:
How about changing the name of the variable?
```suggestion
    let cmd_result  = match opts.subcmd {
```